<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentUserPage</class>
 <widget class="QDialog" name="CurrentUserPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>240</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>当前账户</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/icons/user.png</normaloff>:/icons/user.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_permission">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>*权限*</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>账号</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_modify">
     <property name="text">
      <string>修改信息</string>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/icons/edit_square.png</normaloff>:/icons/edit_square.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_close">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_modify</tabstop>
  <tabstop>pushButton_close</tabstop>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_password</tabstop>
 </tabstops>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>CurrentUserPage</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
